### PR TITLE
Исправление для #15889

### PR DESCRIPTION
**Pull request by lesha724**
Добавлена возможность изменить/дополнить метод setActivePlaceholder после переопределиния базового класса Html

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | #15889

### CODE REVIEW

**Review by samdark — APPROVED** [review]
Please add changelog. Thanks!

**Comment by samdark** [comment]
Merged. Thanks!

File: tests/framework/helpers/HtmlTest.php [hunk 1]
```diff
@@ -1729,6 +1729,34 @@ public function testActiveTextInput_placeholderFillFromModelTabular()
...
+
+        $this->assertContains('placeholder="My placeholder: Name"', $html);
+    }
+}
+
+/**
+ * Class MyHtml
+ * @package yiiunit\framework\helpers
+ */
+class MyHtml extends Html{
```
Comment (rob006) [inline tests/framework/helpers/HtmlTest.php]: Why two classes in single file?

Comment (lesha724) [reply tests/framework/helpers/HtmlTest.php]: Because this class was used only once. If you want I can transfer it to a separate file.

Comment (rob006) [reply tests/framework/helpers/HtmlTest.php]: It breaks PSR-1. I'm for fixing it (including classes existing before this PR).

**Pull request merged**